common/app/views/support/EmailHelpers.scala

```diff
@@ -50,7 +50,7 @@ object EmailHelpers {
   }
 
   def icon(name: String) = Html {
-    s"""<img src="${Static(s"images/email/icons/$name.png")}" class="float-left icon icon-$name">"""
+    s"""<img src="${Static(s"images/email/icons/$name.png")}" class="icon icon-$name">"""
```

Comment:
yeah it is mad, it seems like Gmail goes to the effort of actually stripping out SVGs from the email HTML before allowing it to be rendered in someone's inbox!